src/mappings/lpTokenPriceHandler.ts

```diff
@@ -3,18 +3,25 @@ import { u32 } from '@polkadot/types';
 import { divDecs } from './util';
 
 const relayAssetId = (api.consts.crowdloans.relayCurrency as u32).toNumber();
+const nativeAssetId = api?.consts?.currencyAdapter?.getNativeCurrencyId?.toString();
```

Comment:
what if nativeAssetId undefined

Reply:
This is a copy-paste problem and the question mark can be removed from the middle. The chain should be guaranteed to have this storage